src/main/java/io/pakland/mdas/githubstats/application/FetchPullRequestsFromRepository.java

```diff
@@ -0,0 +1,20 @@
+package io.pakland.mdas.githubstats.application;
+
+import io.pakland.mdas.githubstats.application.dto.PullRequestDTO;
+import io.pakland.mdas.githubstats.application.exceptions.HttpException;
+import io.pakland.mdas.githubstats.domain.repository.PullRequestExternalRepository;
+
+import java.util.List;
+
+public class FetchPullRequestsFromRepository {
+
+    private PullRequestExternalRepository pullRequestExternalRepository;
+
+    public FetchPullRequestsFromRepository(PullRequestExternalRepository pullRequestExternalRepository) {
+        this.pullRequestExternalRepository = pullRequestExternalRepository;
+    }
+
+    public List<PullRequestDTO> execute(Integer repositoryOwnerId, Integer repositoryId) throws HttpException {
+        return this.pullRequestExternalRepository.fetchPullRequestsFromRepository(repositoryOwnerId, repositoryId);
+    }
```

Comment:
As we have said, this is not working since we cannot fetch information from the API using ID's, we must use name/slugs.

Nonetheless, this issue will be fixed in #67.